src/main/java/frc/robot/subsystems/shooterSubsystem/ShooterCommandFactory.java

```diff
@@ -14,8 +14,8 @@
 public class ShooterCommandFactory {
   private final ShooterSubsystem subsystem;
   private ShuffleboardTab shooterTab = Shuffleboard.getTab("ShooterSubsystem");
-  private GenericEntry targetRPM = shooterTab.add("Target RPM", 3050).getEntry();
-  private GenericEntry rightTargetRPM = shooterTab.add("right RPM", 2250).getEntry();
+  private GenericEntry targetRPM = shooterTab.add("Target RPM", 3050 * 1.125).getEntry();
```

Comment:
Is this still being used?  If so, let's remove the 1.125, and put 3050 in constants.  

If not being used, let's remove it.
If used outside of competition, let's put it in an isCompetition block